external-plugins/test-subset/plugin/handler/handler.go

```diff
@@ -0,0 +1,371 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	k8s_v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "sigs.k8s.io/prow/pkg/client/clientset/versioned/typed/prowjobs/v1"
+	"sigs.k8s.io/prow/pkg/config"
+	gitv2 "sigs.k8s.io/prow/pkg/git/v2"
+	"sigs.k8s.io/prow/pkg/github"
+	"sigs.k8s.io/prow/pkg/pjutil"
+
+	pi_github "kubevirt.io/project-infra/robots/pkg/github"
+)
+
+var log *logrus.Logger
+var testSubsetCommentRe = regexp.MustCompile(`^/test-subset (\S+) (\S+|\(.*\))$`)
+
+func init() {
+	log = logrus.New()
+	log.SetOutput(os.Stdout)
+}
+
+type GitHubEvent struct {
+	Type    string
+	GUID    string
+	Payload []byte
+}
+
+type githubClientInterface interface {
+	IsMember(string, string) (bool, error)
+	GetPullRequest(string, string, int) (*github.PullRequest, error)
+}
+
+type loadConfigBytesFunc func(h *GitHubEventsHandler, org, repo string) ([]byte, []byte, error)
+
+var LoadConfigBytesFunc loadConfigBytesFunc = loadConfigBytes
```

Comment:
you have the SetLocal... func, so this variable doesn't need the be exported.

Reply:
I am just saying that the variable doesn't need to be exported because you are setting the default already on init(), where for unit tests you call the SetLocal func, so it can be changed to 
```suggestion
var loadConfigBytesFunc loadConfigBytesFunc = loadConfigBytes
```
as it need not be assignable from outside the package - unless I am missing something.
(I figure you need to change the type name ofcourse)

Reply:
ah right, thanks done